build/llvm.yml

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+trigger: none
+
+variables:
+  CCACHE_DIR: $(Pipeline.Workspace)/ccache
+  CMAKE_INSTALL_PREFIX: $(Pipeline.Workspace)/LLVM
+  INSTALL_LLVM_PACKAGE: true
+  SCCACHE_DIR: $(Pipeline.Workspace)\sccache
+  AQ_LLVM_PACKAGE_GIT_VERSION: ""
+  ENABLE_COMPILER: false
+  ENABLE_VSIX: false
+  ENABLE_LLVM_BUILDS: true
+  AQ_DOWNLOAD_LLVM: false
+
+jobs:
+- job: GetLlvmTag
+  pool:
+    vmImage: 'ubuntu-20.04'
```

Comment:
Strictly 'ubuntu-**20.04**'? (Not 'ubuntu-**latest**'?)
If yes then a code comment answering "why?" would not harm.

Reply:
I think it should work with ubuntu-latest, since on Linux we build with a container using manylinux anyway. So the underlying OS doesn't actually matter.

Reply:
I updated to using "latest" build images for both ubuntu and windows vms.